### PR TITLE
Podperson hair

### PR DESCRIPTION
**Pull request by Sheits**

### CODE REVIEW

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -77,8 +77,30 @@
...
+		"ears" = "None",
+		"frills" = pick(GLOB.frills_list),
+		"spines" = pick(GLOB.spines_list),
+		"body_markings" = pick(GLOB.body_markings_list),
+		"legs" = "Normal Legs", "caps" = pick(GLOB.caps_list),
+		"moth_wings" = pick(GLOB.moth_wings_list),
+		"moth_antennae" = pick(GLOB.moth_antennae_list),
+		"moth_markings" = pick(GLOB.moth_markings_list),
+		"tail_monkey" = "None",
+		"pod_hair" = pick(GLOB.pod_hair_list)
```
Comment (JohnFulpWillard) [inline code/__HELPERS/mobs.dm]: trailing comma

File: code/__HELPERS/mobs.dm [hunk 2]
```diff
@@ -77,8 +77,30 @@
...
+		))
```
Comment (JohnFulpWillard) [inline code/__HELPERS/mobs.dm]: remove a tab here

File: code/modules/client/preferences/species_features/pod.dm [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/proc/generate_pod_side_shots(list/sprite_accessories, key, include_snout = TRUE)
```
Comment (JohnFulpWillard) [inline code/modules/client/preferences/species_features/pod.dm:1]: are you sure this is supposed to be a global proc

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm [hunk 4]
```diff
@@ -9,6 +9,8 @@
 		TRAIT_CAN_STRIP,
 		TRAIT_PLANT_SAFE,
 	)
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/carbon/human/species_types/podpeople.dm:11]: You lack mutant_bodyparts here to mark it being a preference for them

File: tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/species_features.tsx [hunk 5]
```diff
@@ -71,6 +71,10 @@ export const underwear_color: Feature<string> = {
   component: FeatureColorInput,
 };
 
+export const feature_pod_hair: FeatureChoiced = {
```
Comment (JohnFulpWillard) [inline tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/species_features.tsx]: this isn't supposed to be a thing if you want it to be like other features where you have a bunch of icons to choose from, like hair.

**Review by JohnFulpWillard — COMMENTED** [review]
code review

File: code/modules/mob/living/carbon/human/species_types/podpeople.dm (hunk 4 above)
Comment (Sheits) [reply code/modules/mob/living/carbon/human/species_types/podpeople.dm:11]: its an external organ

**Pull request closed without merging**